test/CMakeLists.txt

```diff
@@ -57,6 +57,13 @@ add_custom_target(run-onedpl-tests
     DEPENDS build-onedpl-tests
     COMMENT "Build and run all oneDPL tests")
 
+# Disable checked iterators on Windows
```

Comment:
May be
"Disable the iterator checks on Windows" ? here and on line 64..
I would ask somebody who is a native English speaker...